docs-internal/api/nu-designer-openapi.yaml

```diff
@@ -3350,6 +3402,68 @@ components:
       type: object
     SqlParameterEditor:
       type: object
+    StatisticDto:
+      required:
+      - name
+      - value
+      type: object
+      properties:
```

Comment:
Maybe it's worth, collecting the context of the event, for example, a user on the Firefox browser clicks x times into a button, and on the Safari browser y times and extends it for the NU version. So on version 1.14, we had x clicks, and on version 1.15 y clicks. Also, maybe it's worth having device context, for type (touch device, laptop device), and resolution. 

I know it's a lot of additional data, but IMO, if we are not able to collect them because of query string params length limit, I recon, we should consider changing the approach and find a way to send them, via body as POST method

Reply:
Maybe we should talk about it because it looks like a bigger task? Maybe statistics with variety of devices and resolution will be sufficient? Because if we add next dimension like a device and user context then we will have sessions and if we want implement session then solution like the matomo has big sense.

Reply:
We discussed it internally, and for now, since we won't have a session user concept, we will just collect a count of the events. If we want to get the context like browser, resolution, etc. we will introduce another event type, which will be sent initially on the page open